automated install/basic-install.sh

```diff
@@ -192,10 +192,12 @@ os_check() {
         detected_os="${detected_os_pretty%% *}"
         detected_version=$(cat /etc/*release | grep VERSION_ID | cut -d '=' -f2- | tr -d '"')
 
-        IFS=" " read -r -a supportedOS < <(dig +short -t txt ${remote_os_domain} @ns1.pi-hole.net | tr -d '"')
```

Comment:
Wouldn't it be much simpler to allow `dig` failing instead of adding another variable? Nothing more required to fall back to the already present error message.
```
IFS=" " read -r -a supportedOS < <(dig +short -t txt ${remote_os_domain} @ns1.pi-hole.net | tr -d '"' || :)
```